graphql/schema/types/metadata.graphql

```diff
@@ -3,6 +3,7 @@ input GenerateMetadataInput {
   previews: Boolean!
   markers: Boolean!
   transcodes: Boolean!
+  thumbnails: Boolean!
```

Comment:
This should either be renamed or clarified in a comment that this refers to gallery thumbnails, since this could refer to any number of thumbnail type.

Reply:
added comment